vocode/streaming/telephony/client/vonage_client.py

```diff
@@ -18,33 +23,38 @@ def __init__(self, base_url, vonage_config: VonageConfig):
             private_key=vonage_config.private_key,
         )
         self.voice = vonage.Voice(self.client)
+        self.maybe_aiohttp_session = aiohttp_session
 
     def get_telephony_config(self):
         return self.vonage_config
 
     async def create_vonage_call(
         self, to_phone: str, from_phone: str, ncco: str, digits: Optional[str] = None
     ) -> str:  # returns the Vonage UUID
-        async with aiohttp.ClientSession() as session:
-            async with session.post(
-                f"https://api.nexmo.com/v1/calls",
-                json={
-                    "to": [{"type": "phone", "number": to_phone, "dtmfAnswer": digits}],
-                    "from": {"type": "phone", "number": from_phone},
-                    "ncco": ncco,
-                },
-                headers={
-                    "Authorization": f"Bearer {self.client._generate_application_jwt().decode()}"
-                },
-            ) as response:
-                if not response.ok:
-                    raise RuntimeError(
-                        f"Failed to start call: {response.status} {response.reason}"
-                    )
-                data = await response.json()
-                if not data["status"] == "started":
-                    raise RuntimeError(f"Failed to start call: {response}")
-                return data["uuid"]
+        aiohttp_session = self.maybe_aiohttp_session or aiohttp.ClientSession()
```

Comment:
why not make this a default arg to init?

Reply:
fair, but I'd like to know whether or not to close it